retis/src/core/filters/meta/filter.rs

```diff
@@ -750,7 +750,9 @@ impl<'a> MetaExpr<'a> {
             ));
 
             idx_sz += 1;
-            if idx_sz >= ctx.target.sz {
+            // idx_sz >= ctx.target.sz is just enough, but this is
+            // needed to make old verifier happy.
+            if idx_sz >= std::mem::size_of_val(&ctx.target.md) {
                 break;
```

Comment:
Shouldn't `sz` also checked as my understanding is `sz` can be < `size_of_val(md)`?